src/pages/docs/projects/custom-step-templates.md

```diff
@@ -112,32 +100,24 @@ It's important to note that you can only run script based custom step templates.
 
 To run a script based step template, perform the following.
 
-1. Navigate to **Library ➜ Step templates** area and click the **Run** button next to the script based custom step template or alternately, select a script template and click the **Run** button from the template editor page:
+1. Navigate to **Deploy ➜ Step templates** area and click the **Run** button next to the script based custom step template or alternately, select a script template and click the **Run** button from the template editor page:
 
    ![Run step template](/docs/projects/images/step-templates-run.png)
 
-2. Select a group of targets to run the step on. This can be done by target name or by environments and roles:
-
-   ![Select run targets](/docs/projects/images/step-templates-run-targets.png)
-
-3. Enter any required parameters:
+2. Select a group of targets to run the step on. This can be done by target name or by environments and tags.
+3. Enter any required parameters.
+4. Click the **Run now** button. This will execute the step as a new task. The full script can be found under the Template Parameters tab:
 
-   ![Enter parameter values](/docs/projects/images/step-templates-run-parameters.png)
-
-4. Click the **Run now** button. This will execute the step as a new task and provide the full script:
-
-   ![Task summary](/docs/projects/images/step-templates-run-task-summary.png)
-   ![Task log](/docs/projects/images/step-templates-run-task-log.png)
    ![Task parameters](/docs/projects/images/step-templates-run-task-parameters.png)
 
-To re-run the script against different deployment targets or modify the input parameters, simply click the **Modify and re-run** button.
+To re-run the script against different deployment targets or modify the input parameters, simply click the **Modify and re-run** button in the overflow menu (`...`).
 
 ## Common step properties
 
 All steps have a name, which is used to identify the step.
 
-:::div{.success}
-Be careful when changing names! Octopus commonly uses names as a convenient identity or handle to things, and the steps and actions in a deployment process are special in that way. For example you can use [output variables](/docs/projects/variables/output-variables) to chain steps together, and you use the name as the indexer for the output variable. For example: `#{Octopus.Action[StepA].Output.TestResult}`
```

Comment:
Changed to warning since it says "be careful" which doesn't quite match the success color.